java/src/jmri/jmrix/loconet/SlotManager.java

```diff
@@ -662,6 +662,15 @@ protected boolean checkLackAcceptedBlind(int Byte2) {
         }
     }
 
+    private boolean acceptAnyLACK = false;
+    /**
+     * Indicate that the command station LONG_ACK response details can be ignored
+     * for this operation.  Typically this is used when accessing Loconet-attached boards.
+     */
+    public final void setAcceptAnyLACK() {
+        acceptAnyLACK = true;
+    }
+    
```

Comment:
Provides a default of "false" for acceptAnyLACK, and provides a "setter" which can _only_ set to true.  So you do some `LOCONETOPSBOARD` programming, and perhaps everything works out ok.  What happens when you later try to program a device which relies upon the command station response?  This code would seem to "accept any LONG_ACK" message.  Is that acceptable when working with, say, the programming track?

I _think_ this P/R needs a generic setter with parameter, and that the LnOpsModeProgrammer needs to _return_ the value back to `false` after completing any `LOCONETOPSBOARD` programming.

Reply:
After I wrote the previous comment, I saw that SlotManager, in the normal flow, will restore acceptAnyLACK _if there is any variety of programming LACK.  That ought to address my concern - so long as there aren't any "abnormal excapes" from the normal flow.

What about the abnormal flows?

One possibility I can think of is when a user attempts to read a `LOCONETOPSBOARD` board that is not addressed the way he/she thinks, via a standalone loconet.  In that case, there won't be any OPC_LACK, and there would not be a set of `acceptAnyLACK` to false.

If I recall correctly, there is a "timeout" in the LocoNet programming mechanisms, and if no response is received, the programmer aborts the operation.  Perhaps that "timeout" mechanism _also_ needs to restore `acceptAnyLACK`.

Reply:
@devel-bobm Good point about handling the missing LACK.  I will address

Reply:
Now sets acceptAnyLACK=false in timeout() to handle case where no LACK is received.